pkg/sink/codec/common/helper.go

```diff
@@ -384,3 +377,42 @@ func SanitizeTopicName(name string) string {
 	}
 	return sanitizedName
 }
+
+// UnsafeBytesToString create string from byte slice without copying
+func UnsafeBytesToString(b []byte) string {
+	return *(*string)(unsafe.Pointer(&b))
+}
+
+// UnsafeStringToBytes create byte slice from string without copying
+func UnsafeStringToBytes(s string) []byte {
+	return *(*[]byte)(unsafe.Pointer(
+		&struct {
+			string
+			Cap int
+		}{s, len(s)},
+	))
+}
+
+// UnsafeStringToBinary create binary string from string without copying
+func UnsafeStringToBinary(s string) string {
```

Comment:
Is this function focus on performance and may be called frequently?
If so, we may be better not to use fmt.Sprintf and += to concat string.